src/views/waterUse/WaterUse.vue

```diff
@@ -37,27 +37,50 @@
           </div>
           <p>First, select water use type (Spring season is the default view)</p>
         </div>
-        <button id="te" class="waterUseButton activeParameter" @click="useButtonClick($event)">
+        <button
```

Comment:
when I built the project it was complaining about this needing modifications, so I did the lint fix suggested in the commit message and these changes came through.